tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -635,11 +624,34 @@ function main() {
 
 		case "$action" in
 		install)
+			if [[ "$runtime" =~ ^(k3s|k3s-agent|rke2-agent|rke2-server)$ ]]; then
+			       if [ ! -f "$containerd_conf_tmpl_file" ] && [ -f "$containerd_conf_file" ]; then
+				       cp "$containerd_conf_file" "$containerd_conf_tmpl_file"
+			       fi
+			       # Only set the containerd_conf_file to its new value after
+			       # copying the file to the template location
+			       containerd_conf_file="${containerd_conf_tmpl_file}"
+			       containerd_conf_file_backup="${containerd_conf_tmpl_file}.bak"
```

Comment:
`containerd_conf_file_backup` was already set to the same value at line 604.

Reply:
Indeed, I dropped it from the actions specific code. Thanks!